src/main/webapp/scripts/map.js

```diff
@@ -188,19 +196,54 @@ function placeMarkerAndPanTo(latLng, map) {
 
 /* Function to place markers from the datastore */
 function placeMarker(markerDetails, map) {
-  new google.maps.Marker ({
+  const marker = new google.maps.Marker ({
     position: new google.maps.LatLng(markerDetails.lat, markerDetails.lng),
     map: map,
-    icon: greyIcon, // TODO change the colour of the icon depending on the category
+    icon: getColourMarker(formatCategory(markerDetails.category)),
+    id: markerDetails.id
+  });
+
+  /* Add Listener for Click on Marker */
+   marker.addListener("click", () => {
+     fetchRecommendationInfo(marker.id);
+   });
+}
+
+/* Get recommendation from datastore and fill html */
+function fetchRecommendationInfo(id) {
+  const params = new URLSearchParams();
```

Comment:
Please delete

Reply:
Can you please push that change before merging? :)

Reply:
Done! Sorry, I must have pushed just after I requested your review :)